src/lib/features/scheduler/schedule-services.ts

```diff
@@ -150,4 +150,12 @@ export const scheduleServices = async (
         minutesToMilliseconds(3),
         'updateAccountLastSeen',
     );
+
+    schedulerService.schedule(
+        featureToggleService.setFeatureCreatedByUserIdFromEvents.bind(
+            featureToggleService,
+        ),
+        secondsToMilliseconds(5),
```

Comment:
Every 5 seconds? More often/seldom? If we have a few to get through we should probably make sure they happen at a decent pace

Reply:
I think we can do this every half an hour or so. It shouldn't take that long.

Reply:
As discussed I set it to every 15mins